tests/distributions/test_multivariate.py

```diff
@@ -2121,11 +2121,11 @@ def ref_rand(size, n, eta):
             size=1000,
         )
 
-    @pytest.mark.parametrize(argnames="n", argvalues=[2, 3], ids=["n=2", "n=3"])
-    def test_default_transform(self, n):
+    def test_default_transform(self):
         with pm.Model() as m:
-            pm.LKJCorr("x", n=n, eta=1)
-        m.logp()
+            x = pm.LKJCorr("x", n=2, eta=1, shape=(3, 2))
+        assert isinstance(m.rvs_to_transforms[x], MultivariateIntervalTransform)
```

Comment:
You'll need to import the Transform

Reply:
hahe sure!

Reply:
added in https://github.com/pymc-devs/pymc/pull/7065/commits/9b4bf2acf82798d13cf3d68dbcade82d5dfca765